source/CardGames.py

```diff
@@ -480,18 +653,69 @@ def winnnerCheck(self, house, player):
         print("You won!")
         betting_box.collect(winner)
         print(f"You now have ${player_name.money}")
+
+        screen.fill((background))
+        handsDisplay(False)#blits the cards to the screen
+        resultText = smallfont.render( "You won!", True , color)
+        screen.blit(resultText, (320,300))
+        screen.blit(promptText, (320,350))
+        walletText = smallfont.render("reserve: $"+str(player_name.money),True, color)
+        screen.blit(walletText, (12,6))
+        pygame.display.update()
+        wait()
       elif winner == 1:
         print("You lost. The dealer won!")
         betting_box.collect(winner)
         print(f"You now have ${player_name.money}")
+
+        screen.fill((background))
+        handsDisplay(False)#blits the cards to the screen
+        resultText = smallfont.render( "You lost!", True , color)
+        screen.blit(resultText, (320,300))
+        screen.blit(promptText, (320,350))
+        walletText = smallfont.render("reserve: $"+str(player_name.money),True, color)
+        screen.blit(walletText, (12,6))
+        pygame.display.update()
+        wait()
       else:
         print("It's a push!")
         betting_box.collect(winner)
         print(f"You now have ${player_name.money}")
 
+        screen.fill((background))
+        handsDisplay(False)#blits the cards to the screen
+        resultText = smallfont.render( "It's a push!", True , color)
+        screen.blit(resultText, (320,300))
+        screen.blit(promptText, (320,350))
+        walletText = smallfont.render("reserve: $"+str(player_name.money),True, color)
+        screen.blit(walletText, (12,6))
+        pygame.display.update()
+        wait()
+
     """"Clear hands and ask if they want to play again"""
     print("Cards remaining in deck:", deck.size)
-    start = input("\nWould you like to play again (y/n)? ")
+  
+    screen.fill(background)
+    resultText = smallfont.render( "enter to continue. space to quit", True , color)
+    screen.blit(resultText, (160,300))
+    walletText = smallfont.render("reserve: $"+str(player_name.money),True, color)
+    screen.blit(walletText, (12,6))
+    pygame.display.update()
+
+    waiting = True
+    while waiting:
+      for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+          pygame.quit()
+            
```

Comment:
I think we should reduce the number of decisions the player has to make in order to play the next hand. We shouldn't explicitly ask them if they want to quit. That option can be there when they are inputting a bet, but by default, we should assume they want to bet. 

Perhaps instead of clearing the screen and adding new blit objects, we just blit new objects on the existing screen. For example, after the hand is completed the option for the next bet appears and seemingly transitions to the next game.

Reply:
I removed the quit prompt. It was in there just because I was trying to faithfully upgrade the existing UI structure.